web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/setting-environment-variables.md

```diff
@@ -20,3 +20,48 @@ For example:
 ```
 WPGRAPHQL_URL=https://my-wordpress-site.pantheon.site/wp/graphql
 ```
+
+## Connecting to Multidev Environments
+
+To connect to a Multidev environment, the following helper environment variable
+can be used inside of `gatsby-config.js`.
+
+- `IS_LIVE_ENVIRONMENT` - True if `PANTHEON_ENVIRONMENT_URL` is live.
+
+Either the `PANTHEON_CMS_ENDPOINT` or `WPGRAPHQL_URL` will need to be set.
+
+The `PANTHEON_CMS_ENDPOINT` can be mocked for local development by defining it
+in the `.env.development.local`.
+
+```
+PANTHEON_CMS_ENDPOINT=dev-my-gatsby-site.pantheonsite.io
```

Comment:
```suggestion
PANTHEON_CMS_ENDPOINT=dev-my-wordpress-site.pantheonsite.io
```
Suggest changing `gatsby` to `wordpress` here to clarify that this is the CMS site